.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,74 @@
+Based on the nature of your change, please use the relevant template below and modify to suit your needs.
```

Comment:
Oh, so your issue isn't with the paragraph below? We can just delete this line, then, since I've updated the contribution guide.